code/modules/mining/mine_items.dm

```diff
@@ -224,36 +224,67 @@
 	start_turf = get_turf(src.loc)
 
 	//Fill it
+
+	//The door
 	cur_turf = locate(start_turf.x, start_turf.y-2, start_turf.z)
-	new /obj/machinery/door/airlock/glass(cur_turf)
+	new /obj/machinery/door/airlock/survival_pod(cur_turf)
+
 
+	//Bed middle right
 	cur_turf = locate(start_turf.x+1, start_turf.y, start_turf.z)
-	new /obj/structure/table/wood(cur_turf)
-	new /obj/item/weapon/storage/pill_bottle/dice(cur_turf)
+	new /obj/structure/bed/pod(cur_turf)
+	new /obj/item/weapon/bedsheet/brown(cur_turf)
```

Comment:
I'm thinking purple mostly because mining coloring is purple/brown and this already has brown on the walls.
~theme~